src/devices/mixer.cpp

```diff
@@ -28,51 +28,60 @@ Mixer::Mixer(uint8_t device_type, uint8_t device_id, uint8_t product_id, const s
     : EMSdevice(device_type, device_id, product_id, version, name, flags, brand) {
     LOG_DEBUG(F("Adding new Mixer with device ID 0x%02X"), device_id);
 
-    if (flags == EMSdevice::EMS_DEVICE_FLAG_MMPLUS) {
-        if (device_id <= 0x27) {
-            // telegram handlers 0x20 - 0x27 for HC
-            register_telegram_type(device_id - 0x20 + 0x02D7, F("MMPLUSStatusMessage_HC"), true, MAKE_PF_CB(process_MMPLUSStatusMessage_HC));
-        } else {
-            // telegram handlers for warm water/DHW 0x28, 0x29
-            register_telegram_type(device_id - 0x28 + 0x0331, F("MMPLUSStatusMessage_WWC"), true, MAKE_PF_CB(process_MMPLUSStatusMessage_WWC));
-        }
-    }
+    if (flags == EMSdevice::EMS_DEVICE_FLAG_MP) {
```

Comment:
For HA the id should be added, also HA-config for TAG_NONE.

Reply:
Like `register_device_value(TAG_MP, ...` and setting TAG_MP to the next enum?

Reply:
No, i mean in publish_ha_config for `tpl` and `topic` a third case, like:
```
    if (type_ == Type::HC) {
        snprintf(tpl, sizeof(tpl), "{{value_json.hc%d.id}}", device_id() - 0x20 + 1);
    } else if (type_ == Type::WWC) {
        snprintf(tpl, sizeof(tpl), "{{value_json.wwc%d.id}}", device_id() - 0x28 + 1);
    } else {
        snprintf(tpl, sizeof(tpl), "{{value_json.id}}");
    }
```